qiskit/transpiler/passes/scheduling/padding.py

```diff
@@ -205,11 +232,307 @@ def _pad(
         self,
         dag: DAGCircuit,
         qubit: Qubit,
-        time_interval: int,
+        t_start: int,
+        t_end: int,
         next_node: DAGNode,
         prev_node: DAGNode,
     ):
         if not self.fill_very_end and isinstance(next_node, DAGOutNode):
             return
 
+        time_interval = t_end - t_start
         dag.apply_operation_back(Delay(time_interval, dag.unit), [qubit])
+
+
+class DynamicalDecoupling(BasePadding):
```

Comment:
can you put this class in its own file? i prefer passes to have individual files for better discoverability. it can still be a subclass of another pass.

Reply:
Done in https://github.com/Qiskit/qiskit-terra/pull/7760/commits/54b6f0cabd164b6115469437525d745ae9ce1a6a

Reply:
I just meant split the file `padding.py` into a directory `passes/scheduling/padding`, and put `base_padding.py` and `pad_delay.py` and `dynamical_decoupling.py` as 3 different files in there.

Reply:
Done in https://github.com/Qiskit/qiskit-terra/pull/7760/commits/19d235b8f1d6ad6c54741936367037a058f9b065